### ISSUE TITLE
meross_lan: Setting last_reset for entities with state_class other than 'total' is not supported.

### ISSUE THREAD

**Issue by purcell-lab** [krahabb/meross_lan#163]
## Version of the custom_component

v2.4.3
## Configuration

Auto Configuration

## Describe the bug
WIth the update to core-2022.2.0b0 the meross_lan custom component is reporting issues with state_class

## Debug log

```text

Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:317
Integration: Sensor (documentation, issues)
First occurred: 10:49:05 PM (2 occurrences)
Last logged: 10:49:05 PM

Entity sensor.fridge_energy (<class 'custom_components.meross_lan.sensor.MerossLanSensor'>) with state_class total_increasing has set last_reset. Setting last_reset for entities with state_class other than 'total' is not supported. Please update your configuration if state_class is manually configured, otherwise report it to the custom component author.
Entity sensor.microwave_energy (<class 'custom_components.meross_lan.sensor.MerossLanSensor'>) with state_class total_increasing has set last_reset. Setting last_reset for entities with state_class other than 'total' is not supported. Please update your configuration if state_class is manually configured, otherwise report it to the custom component author.

```

**Comment by krahabb** [comment]
Hello, thank you for signaling this I'll get back at code as sson as possible

**Comment by a-waider** [comment]
+1

The issue still exists in the core-2022.2.0 update.

**Comment by enricoso** [comment]
Hello, will fixed code be available in next release (2.4.4)? Thank you for your precious work.
